mapusaurus/censusdata/views.py

```diff
@@ -23,12 +193,18 @@ def race_summary_as_json(request_dict):
 
     data = {}
     for stats in records:
+        if stats.total_pop:
+            MINPERC = 1.0 * (stats.total_pop - stats.non_hisp_white_only) / stats.total_pop
```

Comment:
i'd vote for storing total minority percentage in db -- it was requested as a standard value for tables